app/router.options.ts

```diff
@@ -0,0 +1,17 @@
+import type { RouterConfig } from '@nuxt/schema'
+
+export default {
```

Comment:
this is what I was looking for few days back, how to do

Reply:
Yeah. It was quite hidden in docs.